src/ostorlab/cli/scan/list/list.py

```diff
@@ -52,6 +55,16 @@ def list_scans(ctx: click.core.Context, page: int, elements: int) -> None:
             console.error("Error fetching scan list.")
 
 
+def _get_risk_rating(risk: Union[risk_rating.RiskRating, str, None]) -> str:
```

Comment:
```suggestion
def _get_risk_rating(risk: risk_rating.RiskRating | str | None) -> str:
```

Reply:
this is oxo man, we have workflow for python 3.9